manifests/g/Google/GooglePlayGames/Beta/1.3.36.152/Google.GooglePlayGames.Beta.locale.en-US.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Google.GooglePlayGames.Beta
+PackageVersion: 1.3.36.152
+PackageLocale: en-US
+Publisher: Google LLC
+PublisherUrl: https://play.google.com/googleplaygames

```

```suggestion
PublisherUrl: https://www.google.com
PublisherSupportUrl: https://support.google.com/
PrivacyUrl: https://policies.google.com/privacy
```
Metadata from previous manifest.